plugins/services/src/js/service-configuration/PodStorageConfigSection.js

```diff
@@ -27,7 +27,7 @@ class PodStorageConfigSection extends React.Component {
         prop: "type"
       },
       {
-        heading: "Size",
+        heading: "Size (MiB)",
```

Comment:
It might not be true that the unit is MiB?

Reply:
External volumes aren't MiB

Reply:
yup, we should append the unit to every row 👍

Reply:
@Poltergeist are external volumes an option for pods? In the dropdown not appearing. If they aren't, then all others are MiB and we can keep it here.

Reply:
@Poltergeist @ahoskins Did you find an answer to this question?

Reply:
Haha no I did not 🙃

Reply:
so is this wrong right now? then we should block this PR 🙈

Reply:
so is this wrong right now? then we should block this PR 🙈
feel free to dismiss my block once you verified this is correct.

Reply:
External volumes are not available for pods so far. This may change in the future. So we may keep this for now.